File: C++/1785.cpp
#include <iostream>
#include <string>

int main() {
  int count;
  std::cin >> count;

  std::string result;
  if (count < 5) {
    result = "few";
  } else if (count < 10) {
    result = "several";
  } else if (count < 20) {
    result = "pack";
  } else if (count < 50) {
    result = "lots";
  } else if (count < 100) {
    result = "horde";
  } else if (count < 250) {
    result = "throng";
  } else if (count < 500) {
    result = "swarm";
  } else if (count < 1000) {
    result = "zounds";
  } else {
    result = "legion";
  }
  std::cout << result << '\n';
  return 0;
}

File: Python/1785.py
ranges = [
    (5, 'few'), (10, 'several'), (20, 'pack'), (50, 'lots'),
    (100, 'horde'), (250, 'throng'), (500, 'swarm'),
    (1000, 'zounds'), (float('inf'), 'legion')
]

count = int(input())
result = next(name for limit, name in ranges if count < limit)
print(result)
